src/components/cohorts/CohortPage.js

```diff
@@ -42,13 +42,20 @@ const CohortPage = () => {
     navigate(`../profile/${userId}`, { replace: true })
   }
 
+  const startDate = cohort?.startDate.split('T')[0]
```

Comment:
This looks like it should be extracted to a helper function in a utility directory